pkgs/development/mobile/androidenv/build-app.nix

```
@@ -9,7 +9,7 @@
 assert release -> keyStore != null && keyAlias != null && keyStorePassword != null && keyAliasPassword != null;
 
 let
-  androidSdkFormalArgs = builtins.functionArgs composeAndroidPackages;
+  androidSdkFormalArgs = composeAndroidPackages.__functionArgs or (builtins.functionArgs composeAndroidPackages);
```

```suggestion
  androidSdkFormalArgs = lib.functionArgs composeAndroidPackages;
```

`lib.functionArgs` does this for us
https://github.com/NixOS/nixpkgs/blob/c9e26d7d3a86e78971ba3588623a9c61d4ffd262/lib/trivial.nix#L434-L443